### ISSUE TITLE
Pipe deprecation message not into xsd file

### ISSUE THREAD

**Issue by simonschaufi** [TYPO3/Fluid.SchemaGenerator#13]
I'm generating the xsd file for fluid with TYPO3 10 and get the following deprecation warning also in the first line of the xsd file:

`Deprecated: TYPO3\CMS\Fluid\ViewHelpers\Widget\AutocompleteViewHelper will be removed in TYPO3 v11.`

How can i pipe it to something else other then the xsd file?

```bash
php vendor/bin/generateschema 'TYPO3\CMS\Fluid' > fluid.xsd
```

**Comment by NamelessCoder** [comment]
The "deprecated" line is actually an annotation where the `@` is stripped off - technically this is considered part of the PHPDOC comment text and cannot be separated from the "normal" text of the comment.

What is the specific use case you are trying to achieve?

**Comment by simonschaufi** [comment]
@NamelessCoder The deprecation message should not be in the xsd file because that line invalidates the xml

**Comment by lolli42** [comment]
This has been resolved in 2.2.0 since that version does not instantiate VH's anymore, so any trigger_error() calls within __construct() of VH's won't make it neither to CLI nor to .xsd files anymore:

https://github.com/TYPO3/Fluid.SchemaGenerator/commit/d5497a09e9e4f55c7a5d2eb2e96ce65d4673edf5
